platform/golden/cfg/debian-11-preseed.cfg

```diff
@@ -0,0 +1,92 @@
+####################
+## General defaults
+
+d-i debian-installer/locale string en_US.UTF-8
+d-i keyboard-configuration/xkb-keymap select us
+d-i pkgsel/install-language-support boolean false
+
+####################
+## Network defaults
+
+d-i netcfg/choose_interface select auto
+d-i netcfg/get_hostname string golden
+d-i netcfg/get_domain string local
+
+
+###################
+## Mirror defaults
+
+d-i mirror/country string manual
+d-i mirror/http/hostname string http.us.debian.org
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+d-i apt-setup/use_mirror boolean true
+
+#################
+## Time defaults
+
+d-i clock-setup/utc boolean true
+d-i time/zone string UTC
+d-i clock-setup/ntp boolean true
+
+##################
+## Login defaults
+
+d-i passwd/root-login boolean false
+d-i passwd/user-fullname string Artificial Wisdom
+d-i passwd/username string packer
+d-i passwd/user-password password packer
+d-i passwd/user-password-again password packer
+d-i passwd/user-default-groups string users,sudo
+
+###################
+## Select packages
+
+tasksel tasksel/skip-tasks  string standard
+tasksel tasksel/first multiselect openssh-server
+popularity-contest popularity-contest/participate boolean false
+
+d-i pkgsel/include string curl wget openssh-server vim sudo cloud-init
```

Comment:
would you consider using neovim instead? Do we really require cloud-init?

Reply:
RE: vim, I really am not particular, but I find even just generic `vi` is adequate.  if `neovim` is better, happy to change up.

RE: cloud-init, happy to remove, I had it there because it seemed it might have been needed to deal with setting passwords, which I've since found a better method for resolving.